iphone/Maps/LocalizedStrings/en.lproj/Localizable.strings

```diff
@@ -1152,11 +1152,11 @@
 
 "religious_places" = "Religious places";
 
-"transit_not_found" = "Subway navigation in this region is not available yet";
+"transit_not_found" = "Metro navigation in this region is not available yet";
 
-"dialog_pedestrian_route_is_long_header" = "Subway route is not found";
+"dialog_pedestrian_route_is_long_header" = "Metro route is not found";
```

Comment:
Either en-US.plroj folder should be added (not sure if it's supported on iOS, testing is required), or default en should be Subway.

Reply:
en-US is the default language, isn't it?